### ISSUE TITLE
When training, how should I do with groundtruth? such as normalization

### ISSUE THREAD

**Issue by zhangshaoyong** [iro-cp/FCRN-DepthPrediction#35]
hello, I am trying to recreate your results on the NYU_depth dataset with Tensorflow, I use your published test code directly as a build network. When training, I found the value of the network output is not between 0 and 1. So I don't know deal with groundtruth to calculate loss. Can you help me to solve this problem?
Thank you.

**Comment by chrirupp** [comment]
We predict depth in meters. No normalization needed.

**Comment by zhangshaoyong** [comment]
@chrirupp when training, I am sure the pixel value of network input(RGB) is between 0 and 255. Is the pixel value of ground truth (depthmap)  between 0 and 255?

**Comment by chrirupp** [comment]
input is between 0 and 255, output between 0 meters and 10 meters
